### PR TITLE
Chore: Typos and grammar

### PR DESCRIPTION
**Pull request by AllanOliveiraM**
Hi!

Working with the library and studying the source code, and PRs I've noticed some super simple grammatical errors, but there are some typos as well. Maybe this will help keep the code even cleaner and more correct :)

### CODE REVIEW

File: test/use-swr-immutable.test.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ describe('useSWR - immutable', () => {
     // mount <Component/> by clicking the button
     fireEvent.click(screen.getByText('mount component'))
 
-    // wait for rerender
+    // wait for rerenderinging
```
Comment (huozhi) [inline test/use-swr-immutable.test.tsx:55]: we use rerender as a noun here which implies the behavior, same as below
```suggestion
    // wait for rerender
```

File: test/use-swr-key.test.tsx [hunk 2]
```diff
@@ -35,11 +35,11 @@ describe('useSWR - key', () => {
     await screen.findByText('data:short request')
 
     await act(() => sleep(100)) // wait 100ms until "long request" finishes
-    screen.getByText('data:short request') // should be "short request" still
+    screen.getByText('data:short request') // should be a "short request" still
```
Comment (huozhi) [inline test/use-swr-key.test.tsx:38]: "short request" is the content, not "a short request", same as below
```suggestion
    screen.getByText('data:short request') // should be "short request" still
```

**Comment by AllanOliveiraM** [comment]
Looking again, I agree with your suggestions. I used grammarly.com and from the looks of it, there are still a lot of nuances in the suggestions that aren't perfect :|

File: examples/server-render/README.md [hunk 3]
```diff
@@ -27,6 +27,6 @@ npm run dev
 
 ## The Idea behind the Example
 
-This examples show how to combine Next.js getServerSideProps with the SWR `fallbackData` option to support Server-Side Rendering.
+These examples show how to combine Next.js getServerSideProps with the SWR `fallbackData` option to support Server-Side Rendering.
```
Comment (huozhi) [inline examples/server-render/README.md:30]:
```suggestion
This example shows how to combine Next.js getServerSideProps with the SWR `fallbackData` option to support Server-Side Rendering.
```

File: mutation/index.ts [hunk 4]
```diff
@@ -49,8 +49,8 @@ const mutation = (<Data, Error>() =>
         // Disable cache population by default.
         const options = Object.assign({ populateCache: false }, config, opts)
 
-        // Trigger a mutation, also track the timestamp. Any mutation that happened
-        // earlier this timestamp should be ignored.
+        // Trigger a mutation, and also track the timestamp. Any mutation that happened
+        // earlier this timestamp should been ignored.
```
Comment (huozhi) [inline mutation/index.ts:53]:
```suggestion
        // earlier this timestamp should be ignored.
```

File: test/type/trigger.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export function useTrigger() {
     (_, opts: { arg: number }) => String(opts.arg)
   )
 
-  // The argument of trigger should be number or undefined.
+  // The argument of the trigger should be number or undefined.
```
Comment (huozhi) [inline test/type/trigger.ts:27]:
```suggestion
  // The argument of `trigger` should be number or undefined.
```

File: core/use-swr.ts [hunk 6]
```diff
@@ -211,17 +211,17 @@ export const useSWRHandler = <Data = any, Error = any>(
     if (getConfig().isPaused()) return false
 
     // Under suspense mode, it will always fetch on render if there is no
-    // stale data so no need to revalidate immediately on mount again.
+    // stale data so no need to revalidate immediately on the mount again.
```
Comment (huozhi) [inline core/use-swr.ts:214]:
```suggestion
    // stale data so no need to revalidate immediately mount it again.
```

**Review by huozhi — APPROVED** [review]

**Pull request merged**